### PR TITLE
Troubleshooting added to check available components in the collector

### PR DESCRIPTION
**Pull request by nerudadhich**
Fix for issue - https://github.com/open-telemetry/opentelemetry.io/issues/2072

### CODE REVIEW

File: content/en/docs/collector/troubleshooting.md [hunk 1]
```diff
@@ -15,6 +15,172 @@ network issues, it can be helpful to send a small amount of data to a collector
 configured to output to local logs. For details, see
 [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#local-exporters).
 
+## Check available components in the collector
```
Comment (tiffany76) [inline content/en/docs/collector/troubleshooting.md:18]:
```suggestion
## Check available components in the Collector
```

**Review by tiffany76 — COMMENTED** [review]
I've made a few small suggestions to help with clarity and conform to the style guide. 

The sample output seems a little long to me. Maybe showing only a couple of each type of component would be clearer?  The Comms folks will know best.

**Comment by nerudadhich** [comment]
> I've made a few small suggestions to help with clarity and conform to the style guide.
> 
> The sample output seems a little long to me. Maybe showing only a couple of each type of component would be clearer? The Comms folks will know best.

Thanks for comment, I agree with sample output. Let's wait for more inputs.

File: content/en/docs/collector/troubleshooting.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/collector/troubleshooting.md]: @theletterf do I remember correctly that we agreed that we write "Collector" if we are referring to the OpenTelemetry Collector?

**Review by mx-psi — COMMENTED** [review]
Should we note that the output format may change across versions?

**Comment by nerudadhich** [comment]
> Should we note that the output format may change across versions?

I have updated the sample output and added the note as well. Please review again and confirm if anything else need to add/update.

**Review by mx-psi — APPROVED** [review]

**Review by tiffany76 — COMMENTED** [review]
LGTM!

**Review by theletterf — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Pull request merged**